rstudio/c9s-python-3.9/Dockerfile

```diff
@@ -77,10 +77,12 @@ RUN yum -y module enable nginx:$NGINX_VERSION && \
     yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
     rpm -V $INSTALL_PKGS && \
     nginx -v 2>&1 | grep -qe "nginx/$NGINX_VERSION\." && echo "Found VERSION $NGINX_VERSION" && \
-    # spawn-fcgi is not in epel9
-    rpm -i --nodocs https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/37/Everything/x86_64/os/Packages/s/spawn-fcgi-1.6.3-23.fc37.x86_64.rpm && \
+    # Install supervisord
+    yum -y install supervisor && \
```

Comment:
Why is this kept as a separate yum transaction rather going directly to the `INSTALL_PKGS` above?

Reply:
I guess we've never been very careful about minimizing layers in the container. It would not matter on OSBS which squashes them at the end. It does somewhat matter on quay.io. I believed we have a task about fixing this in all images but I did not find it.

Reply:
This is run as one `RUN` so it generates one layer anyway.

But it is confusing for a reader -- what is so special about `supervisor` that it got its own `yum` invocation?